content/webapp/components/SeriesNavigation/SeriesNavigation.tsx

```diff
@@ -33,6 +33,8 @@ const SeriesNavigation: FunctionComponent<Props> = ({ series, items }) => {
         </Space>
       </Layout8>
     </SpacingComponent>
+  ) : (
```

Comment:
I imagine they're equivalent, but I _think_ we've returned `null` from a ternary like this elsewhere, instead of using an empty `Fragment`

Reply:
Will change. It makes sense – no point creating an object you're immediately going to discard when `null` is right there.